### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by joerossi215**

### CODE REVIEW

File: MoodQuiz.php [hunk 1]
```diff
@@ -0,0 +1,98 @@
+<?php 
+   session_start();
+?>
+<!DOCTYPE html>
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
```
Comment (Taraque-A) [inline MoodQuiz.php:7]: Looks nice!

File: MoodQuiz.php [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+	<title>Mood Quiz</title>
+    <link rel="icon" type="image/x-icon" href="favicon.ico">
+	
+    <link rel="stylesheet" type="text/css" href="css/animate.css">
+	
+	<link rel="stylesheet" href="bootstrap.min.css">
+	
+	<link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet">
+	
+	<link rel="stylesheet" type="text/css" href="MoodQuizStyle.css">
```
Comment (Taraque-A) [inline MoodQuiz.php:17]: Looks really good!

File: MoodQuiz.php [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/animejs/2.0.2/anime.min.js"></script>
+</head>
+
+<body>
+    <div id="background-wrap">
+        <div class="x1">
+            <div class="cloud"></div>
```
Comment (Taraque-A) [inline MoodQuiz.php:24]: Class look's perfect!

**Review by yellow-tulips — APPROVED** [review]
Your code looks good to upload!

File: MoodQuiz.php [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+		<h1>Mood Quiz</h1>
+	</div>
+	<div class="container text-center">
+		<ul id="quiz" class="list-group">
+
+		</ul>
+	</div>
+
+
+	<div class="container text-center results hide">
```
Comment (Taraque-A) [inline MoodQuiz.php:76]: Great idea!

**Review by Taraque-A — APPROVED** [review]

**Review by farzanaisrat — APPROVED** [review]
This looks good functionally. Good job!

**Review by tasnhubaislam — APPROVED** [review]
Looks pretty good

**Pull request merged**